packages/config/config/devices/0x0312/n4003.json

```diff
@@ -0,0 +1,56 @@
+{
+	"manufacturer": "NewOne",
+	"manufacturerId": "0x0312",
+	"label": "N4003",
+	"description": "SmartPlug with Energy Monitor",
+	"devices": [
+		{
+			"productType": "0xac01",
+			"productId": "0x4003"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "~/0x0312/templates/newone_template.json#led_indicator_four_options_inverted_load"
+		},
+		{
+			"#": "2",
+			"$import": "~/0x0312/templates/newone_template.json#auto_off_timer"
+		},
+		{
+			"#": "3",
+			"$import": "~/0x0312/templates/newone_template.json#auto_on_timer"
+		},
+		{
+			"#": "4",
+			"$import": "~/templates/master_template.json#state_after_power_failure_prev_off_on"
+		},
+		{
+			"#": "5",
+			"$import": "~/templates/master_template.json#power_wattage_report"
+		}
+		{
+			"#": "6",
+			"$import": "~/templates/master_template.json#power_time_report"
+		}
+		{
+			"#": "7",
+			"$import": "~/templates/master_template.json#power_current_report"
+		}
+		{
+			"#": "8",
+			"$import": "~/templates/master_template.json#power_kwh_report"
+		}
+
+	],
```

Comment:
```suggestion
		{
			"#": "5",
			"$import": "~/templates/master_template.json#power_wattage_report"
		},
		{
			"#": "6",
			"$import": "~/templates/master_template.json#power_time_report"
		},
		{
			"#": "7",
			"$import": "~/templates/master_template.json#power_current_report"
		},
		{
			"#": "8",
			"$import": "~/templates/master_template.json#power_kwh_report"
		}
	],
```

Reply:
How did you edit this? Both VSCode and Gitpod should scream at you for missing a comma in JSON.

Reply:
VSCode. Not sure why I didn't yell 🤷